### DISCUSSION TITLE
Should the resource link in `MedNISTDataset` be updated?

### DISCUSSION THREAD

**Discussion by yiheng-wang-nv** [Project-MONAI/MONAI#3839] [Q&A] [answered]
According to https://github.com/Project-MONAI/MONAI-extra-test-data/releases/tag/0.8.1, should the google drive based link of the MedNIST dataset be updated?

**Comment by yiheng-wang-nv** [comment, marked as answer]
When I checked `tutorials/2d_registration/registration_mednist.ipynb`, I met the following error:

![Screen Shot 2022-02-22 at 1 37 11 PM](https://user-images.githubusercontent.com/68361391/155069706-3baa9403-a8af-481e-a42e-643394eaf19c.png)

The warning message shows:

![Screen Shot 2022-02-22 at 1 40 35 PM](https://user-images.githubusercontent.com/68361391/155070099-d6fc4e5d-c596-4045-bd64-fe7d0ed1b4d4.png)

**Comment by Nic-Ma** [comment]
Hi @yiheng-wang-nv ,

I think we should update to use the new dataset location to avoid the Google Drive limitation.
Could you please help submit a quick PR to update it?
CC @wyli 

Thanks in advance.
